pkg/controller/remoteingress/remoteingress_controller_test.go

```diff
@@ -539,13 +562,24 @@ func testRouteSelector() *metav1.LabelSelector {
 	return selector
 }
 
-func addIngressToClusterDeployment(cd *hivev1.ClusterDeployment, ingressName, ingressDomain string, routeSelector, namespaceSelector *metav1.LabelSelector, servingCertificate string) *hivev1.ClusterDeployment {
+func testHttpErrorCodePages(configmap string) *configv1.ConfigMapNameReference {
+	if configmap != "" {
+		httpErrorCodePages := &configv1.ConfigMapNameReference{
+			Name: configmap,
+		}
+		return httpErrorCodePages
+	}
+	return nil
+}
+
+func addIngressToClusterDeployment(cd *hivev1.ClusterDeployment, ingressName, ingressDomain string, routeSelector, namespaceSelector *metav1.LabelSelector, servingCertificate string, httpErrorCodePages *configv1.ConfigMapNameReference) *hivev1.ClusterDeployment {
```

Comment:
This change seems a bit wasteful. Could simplify and reduce LOC count by just making `testClusterDeploymentWithHttpErrorCodePages()` explicitly add `cd.spec.Ingress[0].HttpErrorCodePages`. Or even do that inline in the test setup struct.